### ISSUE TITLE
How can I set initial number of rows of the v-textarea on the Editor?

### ISSUE THREAD

**Issue by UedaTakeyuki** [DCsunset/vuetify-markdown-editor#14]
Thank you for your great job! This editor is so helpful!

This is one help request, Is there any way to set the initial row number of v-textarea on the Editor from the App?

For example;

```vue
<Editor
  mode="preview"
  ref="editor"

  rows="1"

   hint="Hint"
  :outline="false"
  :render-config="renderConfig"
  v-model="text"
/>
```

Then, **rows="1"**  affect to the initial row number of v-textarea on the Editor as follows:

```vue
<v-textarea
	solo
	flat
	:counter="counter"

	rows="1"
```

Thank you for your kindly help!

**Comment by DCsunset** [comment]
Hello, currently the initial row number cannot be set.
However, I can add this feature in the future.

**Comment by UedaTakeyuki** [comment]
Thank you! I'm looking forward to it!

**Comment by DCsunset** [comment]
Hello @UedaTakeyuki,
I have just released version v3.3.0 that supports the `rows` props.
Besides, nearly all the props in `v-textarea` now can be used directly.

Hope the new release can solve this issue.

**Comment by UedaTakeyuki** [comment]
It works perfectly! Thank you for your great jobs!
Moreover, I've learned how to use $attrs from your code, this solution looks so smart.
